services/consensuscontext/validate_results_block.go

```diff
@@ -20,21 +20,22 @@ import (
 
 type rxValidator func(ctx context.Context, vcrx *rxValidatorContext) error
 
+// TODO v3 get rid of this concept of validator contex. it just unflexible.
```

Comment:
can you either clarify this comment here  or replace it with an issue in github?

Reply:
done